### ISSUE TITLE
remove cdo in nc.ocg_utils.py

### ISSUE THREAD

**Issue by nilshempelmann** [bird-house/eggshell#32]
The `call` function in the `ocg_utils` module uses `cdo` to perform regridding. 
To reduce dependencies the regridding should be done within ocgis or simply removed and performed in a seperate module. 
@nkadygrov @huard Where/How are you using reggridding processes?

**Comment by nkadygrov** [comment]
I agree that using cdo regriding inside ocgis call is, I would say, ridiculous. Especially the way I did it. I had some troubles using ESMF regriding, now it's time to change it.
I use regriding when need to compare models vs reanalysis when they are on different resolution. 
Usually regrid to reanalysis/obs resolution.
Moreover, if I need specific domain, I regrid first full domain to the high-res, and after crop the domain of interest.

**Comment by nilshempelmann** [comment]
Can we disable the regridding option for now in the `eggshell.nc.ocg_utils`?

**Comment by nkadygrov** [comment]
I'm ok with that, I use blackswan.ocgis_module for now.
